testdata/test/contract_test.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require 'json'
+require 'yaml'
+
+module ContractTest
+  class TestMergeConfig < Minitest::Test
+    def test_package_json_contents
+      assert File.exist?('tmp/integration_test/package.json')
+
+      package_json_contents = File.read('tmp/integration_test/package.json')
+      package_json = JSON.parse(package_json_contents)
+      dependencies = package_json['dependencies']
+      dev_dependencies = package_json['devDependencies']
+      scripts = package_json['scripts']
+
+      extension_config_yml = YAML.load_file('testdata/extension.config.integration.yml')
+      expectedRendererVersion = extension_config_yml['extensions'][0]['development']['renderer']['version']
+
+      assert_equal('integration_test', package_json['name'])
+      assert_equal(expectedRendererVersion, dependencies['@shopify/checkout-ui-extensions-react'])
```

Comment:
Made the change suggested here: https://github.com/Shopify/shopify-cli-extensions/pull/289#discussion_r866192234